languages/javascript/concepts/errors/about.md

```diff
@@ -1 +1,67 @@
-TODO: add information on errors concept
+Errors are useful to report when something is wrong or unexpected in a program or a piece of code.
+
+They are javascript objects.
+
+The main property of this object is `message`:
+
+```javascript
+const error = new Error('Oops, something went wrong')
+
+console.log(error.message)
+// => "Oops, something went wrong"
+```
+
+Using the `throw` syntax, you can throw an Error.
+
+```javascript
+throw new Error('Oops')
+```
+
+When an Error is thrown, the current execution is stopped and resume in the first catch block of the call stack.
+
+```javascript
+try {
+  throw new Error('Oops')
+} catch (error) {
+  console.log(error.message)
+  // => "Oops"
+}
+```
+
+As any object in Javascript the Error can be "extended" to create Custom errors. You can use the `instanceof` syntax to check if the error catched is an instance of a particular object.
```

Comment:
Hey, first time jumping in to help out. Should `error catched` be `error caught` ?

Reply:
Indeed, thank you for your review. I'll correct all the mistakes you pointed out 👍  (sorry I'm not a native english speaker, but I do appreciate the review)

Reply:
Great suggestions @pertrai1, thank you very much. 

For future reference, GitHub allows for _suggestions_, which allow the author to apply the suggestion directly, without manually having to copy the work. You can access it by using the insert suggestion button: 
![image](https://user-images.githubusercontent.com/1964376/105856324-6915ff00-5fe9-11eb-9e1c-67aee3042ae6.png)

```suggestion
As any object in Javascript the Error can be "extended" to create Custom errors. You can use the `instanceof` syntax to check if the error caught is an instance of a particular object.
```